PC/msvcrtmodule.c

```diff
@@ -604,7 +604,7 @@ exec_module(PyObject* m)
     INSERTINT(m, "SEM_NOGPFAULTERRORBOX", SEM_NOGPFAULTERRORBOX);
     INSERTINT(m, "SEM_NOOPENFILEERRORBOX", SEM_NOOPENFILEERRORBOX);
 #endif
-#ifdef _DEBUG
+#ifdef Py_DEBUG
```

Comment:
Same as the previous comment - these constants are from the CRT, and `_DEBUG` is the right variable for the CRT.